theme_bookstore/views/snippets/s_striped_centertop.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_striped_centertop" inherit_id="website.s_striped_centertop">
+    <!-- Title -->
+    <xpath expr="//h1" position="replace" mode="inner">
+        A World of Knowledge Awaits
+    </xpath>
+    <!-- Paragraph -->
+    <xpath expr="//p[hasclass('lead')]" position="replace" mode="inner">
+        Immerse yourself in a rich collection of books, magazines, and media that inspire learning, imagination, and creativity.
+    </xpath>
+    <!-- Button -->
+    <xpath expr="//a[hasclass('btn')]/t" position="replace" mode="inner">
+        Browse the Collection
```

Comment:
@xlu-odoo In all the buttons, you uppercase each word even if it's a grammatical word. Is it wanted?